src/logging/printk.c

```diff
@@ -70,6 +70,12 @@ int writek_int(int value){
 	return write_to_log_buff(buf);
 }
 
+int writek_float(float value){
+	char buf[20];
```

Comment:
Yes.  Floating point has up to 7 digits, plut sign and period and exp values in rare cases.